app/Http/Controllers/PasswordResetController.php

```diff
@@ -24,138 +28,112 @@ public function __construct()
         $this->middleware('throttle:20,1440,password-reset:');
```

Comment:
kinda not related, but i noticed this seems quite high. even in the case of a proxy it could probably be lowered considerably..

Reply:
I guess it's okay. Just wondering if it would make sense to make it something like 2 per 2 hours instead, to reduce the ability to spam a heap at once.